6.x/crud-fields.md

```diff
@@ -1401,6 +1401,43 @@ Input preview:
 
 <hr>
 
+<a name="dropzone"></a>
+### dropzone <span class="badge badge-pill badge-info">PRO</span>
```

Comment:
These instructions do not look complete to me. Doesn't this need to look more like this?

https://github.com/Laravel-Backpack/PRO/pull/125#issuecomment-1441730323

Reply:
If I follow these instructions, will the field work? Or is there more information needed here, that I'm not given? Let's aim for a good experience.